lib/Kafka/Librd.pm

```diff
@@ -74,6 +80,12 @@ sub new {
     }
 }
 
+
+sub _log_cb_entry_point {
+    my ($kafka, $level, $facility, $message) = @_;
+    $log_cb->($kafka, $level, $facility, $message);
```

Comment:
I just read the documentation of [rd_kafka_conf_set_log_cb](https://github.com/edenhill/librdkafka/blob/40e40da3c13db0d1933c51d24fc7e204d264512f/src/rdkafka.h#L2012):

> An application MUST NOT call any librdkafka APIs or do any prolonged work in a non-forwarded \c log_cb.

Probably it might be a good idea **not** to forward the `$kafka` variable to the callback function, since it shouldn't be used with API calls anyways.